src/main/resources/static/swagger.yaml

```diff
@@ -262,4 +315,123 @@ components:
           enum: [SUCCEEDED, ERRORED]
         data:
           type: object
-          description: data is a FHIR bundle. https://www.hl7.org/fhir/bundle.html
\ No newline at end of file
+          description: data is a FHIR bundle. https://www.hl7.org/fhir/bundle.html
+    ErrorResponse:
+      type: object
+      properties:
+        error:
+          $ref: '#/components/schemas/Error'
+      xml:
+        name: ErrorResponse
+    Error:
+      type: object
+      properties:
+        code:
+          type: integer
+          enum: [1000, 10001]
+          description: >
+            1. Error code 1000 : No patient found
+            2. Error code 1001: Multiple patients found
+        message:
+          type: string
+      xml:
+        name: Error
+    DataNotification:
+      type: object
+      required:
+        - transactionId
+        - entries
+        - keyMaterial
+      properties:
+        transactionId:
+          type: string
+          format: uuid
+          description: Transaction Id issued when data requested.
+        entries:
+          type: array
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/EntryContent'
+              - $ref: '#/components/schemas/EntryLink'
+        keyMaterial:
+          $ref: '#/components/schemas/KeyMaterial'
+    EntryContent:
+      type: object
+      required:
+        - content
+        - media
+      properties:
+        content:
+          type: string
+          description: Encrypted content
+          example: afafaeqrdfasfasfasfasfasfasdfa
+        media:
+          type: string
+          enum: ['application/fhir+json']
+        checksum:
+          type: string
+        carecontextReference:
+          type: string
+          description: care context reference number.
+    EntryLink:
+      type: object
+      required:
+        - link
+        - media
+      properties:
+        link:
+          type: object
+          required:
+            - href
+          properties:
+            href:
+              type: string
+              description: Encrypted content
+              example: https://data-from.net/sa2321afaf12e13
+            carecontextReference:
```

Comment:
any reason why "carecontextReference" should be inside the link element? also if not, then we don't need an object for link. it can be just "href" or "link". I prefer just "link"

Reply:
also, component wise does it require 2 different object EntryContent, EntryLink .. can't in Swagger API spec, able to specify that only one of the attributes (link or content) be specified? If not, then ok.

Reply:
Having two different component makes sense as we will keep it in the code.

Reply:
> any reason why "carecontextReference" should be inside the link element? also if not, then we don't need an object for link. it can be just "href" or "link". I prefer just "link"
I think that's what we agreed upon